### PR TITLE
Cloudflare.Warp version 24.2.187.0 (microsoft#144983)

### PR DESCRIPTION
**Pull request by paulober**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Does require https://github.com/microsoft/winget-pkgs/pull/144982 to be merged first or update min Webview2Runtime version.

### CODE REVIEW

**Comment by paulober** [comment]
> Does this need a minimum version? Could it be removed as that is what is causing an issue with validation.

Don't know, a previous maintainer added it.
But the dependency it self is not causing the issue but the version I selected. Setting it to the last version would fix the issue, or to wait for the new version to get merged.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-03-17T21:18:23.6031496Z ##[error] One or more errors occurred. (Manifest Error: No Suitable Minimum Version:  [PackageIdentifier] Value: Microsoft.EdgeWebView2Runtime

(Automated response - build 870.)

**Comment by paulober** [comment]
> Automatic Validation ended with:
> 
> > 2024-03-17T21:18:23.6031496Z ##[error] One or more errors occurred. (Manifest Error: No Suitable Minimum Version:  [PackageIdentifier] Value: Microsoft.EdgeWebView2Runtime
> 
> (Automated response - build 870.)

Strange cause the package Microsoft.EdgeWebView2Runtime is in the widget repository. And was dependency of the previous version of this package.

File: manifests/c/Cloudflare/Warp/24.2.187.0/Cloudflare.Warp.installer.yaml
Comment (ItzLevvie) [inline manifests/c/Cloudflare/Warp/24.2.187.0/Cloudflare.Warp.installer.yaml:21]:
```suggestion
```
Removing this will unblock this package.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
[Policy] reset feedback

**Comment by paulober** [comment]
Ok, somehow the cloudflare installer always has different hashes and the AppCenter download is blocked by a token and stuff that can't be but as static url into the manifest.
So maybe this application should be removed from winget-pkgs as the old manifest also doesn't work anymore because of their unified url.

**Comment by stephengillie** [comment]
Hi @paulober,

Yes, unfortunately this might be the best course for this package. It seems like the developer and/or host are trying to limit the number of downloads. Please feel free to open a PR to remove the previous version.

**Pull request closed without merging**
